moveit_configs_utils/moveit_configs_utils/moveit_configs_builder.py

```diff
@@ -217,14 +223,16 @@ def robot_description(
             robot_description_file_path = self.__urdf_package / self.__urdf_file_path
         else:
             robot_description_file_path = self._package_path / file_path
+        mappings = self.__urdf_xacro_args
         if (mappings is None) or all(
             (isinstance(key, str) and isinstance(value, str))
             for key, value in mappings.items()
         ):
             try:
                 self.__moveit_configs.robot_description = {
                     self.__robot_description: load_xacro(
-                        robot_description_file_path, mappings=mappings
+                        robot_description_file_path,
+                        mappings=mappings or self.__urdf_xacro_args,
```

Comment:
Why do we need this `or` when you assigned `mappings` on L226 (does 226 need to be removed like Jafar suggested)?

Reply:
Yeah, I think that line needs to be removed

Reply:
The assignment should be removed now.

Reply:
I thought I cut it out of there but I must have accidentally undo'd that change. Thank you @JafarAbdi for cleaning that up for me